CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 1.2.0 (2022-02-23)
+
+
+### Features
+
+* identify DOM events using CSS locator ([#87](https://github.com/aws-observability/aws-rum-web/issues/87)) ([1c911e0](https://github.com/aws-observability/aws-rum-web/commit/1c911e0bf14841567c8bffe8367fe4b5f4977c9f))
+
+
+### Bug Fixes
+
+* add Cache-Control max-age to CDN files ([#76](https://github.com/aws-observability/aws-rum-web/issues/76)) ([fad8fb9](https://github.com/aws-observability/aws-rum-web/commit/fad8fb9cc39fbd11b5e58501468b0a1971cc4279))
+* capture PerformanceNavigationTiming events even when window.load fires before plugin loads ([#81](https://github.com/aws-observability/aws-rum-web/issues/81)) ([ece1306](https://github.com/aws-observability/aws-rum-web/commit/ece1306d82f51453d4009701eed4051d91708810))
```

Comment:
It seems that we had a couple bug fixes also listed under the previous version as well. Are these duplicates intentional? 

#81 lines 16 & 33
#85 lines 17 & 34
#80 lines 18 and 35

Reply:
These are autogenerated by standard-version, but this seems to be a bug that has been reported years back. Will recreate a commit with the duplicates removed.